source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,181 @@
+# Install Prerequisites
+
+[//]: # (If any commented statements become false, change prior text)
+
+Before you begin, you will need to have access to our Beta. If you don't have access yet, request access at [bons.ai][1] and then come visit us in the [Forums][17] and let us know why you're excited to try out Bonsai!
+
+The next section walks you through how to install Python, Git, and Pip, but if you're already ahead of the curve and have them all installed, skip down to [Setup the Bonsai CLI][2]. 
+
+**Note:** We highly recommend Mac users use [HomeBrew][6] or download Python directly in the General Install instructions so they can perform the following steps on user-land Python rather than [system Python][13].
+
+**If you have Anaconda installed on your system, skip down to [Install with Anaconda][9].**
+
+If you *do not* have Anaconda installed on your system continue to follow our **General Install** instructions.
+
+## General Install
```

Comment:
Consider separating out the instructions to:
 * Windows
 * Mac OS X
 * Windows with chocolatey
 * Mac OS X with Homebrew
 * Anaconda (this assumes there's no difference between windows and mac anaconda)

If there is a difference:
 * Anaconda on Windows
 * Anaconda on Mac OS X

Optionally:
 * Mac OS X using system python and virtualenv
 * Debugging python and pip issues (hopefully this could just be a sentence with a link and not a whole section)